addons/project/views/project_views.xml

```diff
@@ -1333,6 +1417,14 @@
             <field name="name">My Tasks</field>
             <field name="res_model">project.task</field>
             <field name="view_mode">kanban,tree,form,calendar,pivot,graph,activity</field>
+            <field name="view_ids" eval="[Command.clear(),
+                (0, 0, {'view_mode': 'kanban'}),
+                (0, 0, {'view_mode': 'tree'}),
+                (0, 0, {'view_mode': 'calendar', 'view_id': ref('view_task_all_calendar')}),
+                (0, 0, {'view_mode': 'pivot'}),
+                (0, 0, {'view_mode': 'graph'}),
+                (0, 0, {'view_mode': 'activity'}),
+            ]"/>
```

Comment:
Same as above, prefer creating `ir.actions.act_window.view` in such case 🙂

Reply:
Done